docs/user-guide/configuring.md

```diff
@@ -652,14 +651,23 @@ For example, placing the following `.eslintignore` file in the current working d
 ```text
 # /node_modules and /bower_components ignored by default
 
-# Ignore files compiled from TypeScript and CoffeeScript
-**/*.{ts,coffee}.js
-
 # Ignore built files except build/index.js
 build/
 !build/index.js
 ```
 
+### Ignore Patterns
+
+* Brace expansion is not supported in the `.gitignore` specification. An alternative is to match multiple files with multiple patterns. For example, the equivelant of `**/*.{js,ts,coffee}` would be:
+
+```
+# Ignore multiple patterns
+eslint . --ignore-pattern **/*.js --ignore-pattern **/*.ts --ignore-pattern **/*.coffee
+
+# Include multiple extensions
+eslint . --ext js --ext ts --ext coffee
```

Comment:
No problem, happy to remove it. This was what I was having problems with, so I thought I'd mention it